### ISSUE TITLE
Validation accuracy lower than expected

### ISSUE THREAD

**Issue by yushulingfen** [fuqianggu/EventDrop#2]
Hello, thank you for your code. I got a similar result on N-Cars. Then I modified it and tested it on N-Caltech101 that the dataset downloaded form [here](https://github.com/uzh-rpg/rpg_event_representation_learning) , but there seems to be a huge gap between the results of the experiment . Can you send me the code for working with N-Caltech101 dataset and training N-Caltech101 dataset?
![image](https://user-images.githubusercontent.com/102143971/159443073-42d10e29-f9d3-4b08-920c-ad6d40cd0bdc.png)

**Comment by thomhert** [comment]
> 

Hi @yushulingfen 
How did you run the N-Cars dataset? Where did you download the dataset?
I used this dataset https://www.prophesee.ai/2018/03/13/dataset-n-cars/ but the the github code didn't work. I got the error about loading the dataset. Can you share your environments like python and pytorch version? Thank you!

**Comment by chenjiefighting** [comment]
Hello, may I ask how the N-CARS data set is preprocessed? I see that the preprocessing method is not mentioned in this project. Thank you.
